fixtures/dom/src/components/fixtures/number-inputs/index.js

```diff
@@ -27,7 +27,7 @@ function NumberInputs() {
         <NumberTestCase />
 
         <p className="footnote">
-          <b>Notes:</b> Chrome and Safari clear trailing decimals on blur. React
+          <b>Notes:</b> Modern Chrome and Safari {'<='} 6 clear trailing decimals on blur. React
```

Comment:
Dropping decimal places on blur seems to no longer be an issue in modern safari 🎉 .